.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -54,12 +54,14 @@ KAFKA_HOST_PORT=$(kubectl get secret -l app.kubernetes.io/name=backbeat-config,a
+
 # Azure archive tests
 AZURE_ARCHIVE_ACCESS_TIER="Hot"
 AZURE_ARCHIVE_MANIFEST_ACCESS_TIER="Hot"
 
 # Setting CTST world params
-WORLD_PARAMETERS='{"subdomain":"'${SUBDOMAIN}'","ssl":false,"port":"'${ZENKO_PORT}'","AccountName":"'${ZENKO_ACCOUNT_NAME}'","AdminAccessKey":"'${ADMIN_ACCESS_KEY_ID}'","AdminSecretKey":"'${ADMIN_SECRET_ACCESS_KEY}'","VaultAuthHost":"'${VAULT_AUTH_HOST}'","NotificationDestination":"'${NOTIF_DEST_NAME}'","NotificationDestinationTopic":"'${NOTIF_DEST_TOPIC}'","NotificationDestinationAlt":"'${NOTIF_ALT_DEST_NAME}'","NotificationDestinationTopicAlt":"'${NOTIF_ALT_DEST_TOPIC}'","KafkaHosts":"'${KAFKA_HOST_PORT}'","KeycloakPassword":"'${KEYCLOAK_TEST_PASSWORD}'","KeycloakHost":"'${KEYCLOAK_TEST_HOST}'","KeycloakPort":"'${KEYCLOAK_TEST_PORT}'","keycloakRealm":"'${KEYCLOAK_TEST_REALM_NAME}'","keycloakClientId":"'${KEYCLOAK_TEST_CLIENT_ID}'","keycloakGrantType":"'${KEYCLOAK_TEST_GRANT_TYPE}'","StorageManagerUsername":"'${STORAGE_MANAGER_USER_NAME}'","StorageAccountOwnerUsername":"'${STORAGE_ACCOUNT_OWNER_USER_NAME}'","DataConsumerUsername":"'${DATA_CONSUMER_USER_NAME}'","ServiceUsersCredentials":'${SERVICE_USERS_CREDENTIALS}',"azureAccountName":"'${AZURE_ACCOUNT_NAME}'","azureAccountKey":"'${AZURE_SECRET_KEY}'","azureArchiveContainer":"'${AZURE_ARCHIVE_BUCKET_NAME}'","AzureArchiveAccessTier":"'${AZURE_ARCHIVE_ACCESS_TIER}'","azureArchiveManifestTier":"'${AZURE_ARCHIVE_MANIFEST_ACCESS_TIER}'","azureArchiveQueue":"'${AZURE_ARCHIVE_QUEUE_NAME}'"}'
+WORLD_PARAMETERS='{"subdomain":"'${SUBDOMAIN}'","ssl":false,"port":"'${ZENKO_PORT}'","AccountName":"'${ZENKO_ACCOUNT_NAME}'","AdminAccessKey":"'${ADMIN_ACCESS_KEY_ID}'","AdminSecretKey":"'${ADMIN_SECRET_ACCESS_KEY}'","VaultAuthHost":"'${VAULT_AUTH_HOST}'","NotificationDestination":"'${NOTIF_DEST_NAME}'","NotificationDestinationTopic":"'${NOTIF_DEST_TOPIC}'","NotificationDestinationAlt":"'${NOTIF_ALT_DEST_NAME}'","NotificationDestinationTopicAlt":"'${NOTIF_ALT_DEST_TOPIC}'","KafkaHosts":"'${KAFKA_HOST_PORT}'","KeycloakPassword":"'${KEYCLOAK_TEST_PASSWORD}'","KeycloakHost":"'${KEYCLOAK_TEST_HOST}'","KeycloakPort":"'${KEYCLOAK_TEST_PORT}'","keycloakRealm":"'${KEYCLOAK_TEST_REALM_NAME}'","keycloakClientId":"'${KEYCLOAK_TEST_CLIENT_ID}'","keycloakGrantType":"'${KEYCLOAK_TEST_GRANT_TYPE}'","StorageManagerUsername":"'${STORAGE_MANAGER_USER_NAME}'","StorageAccountOwnerUsername":"'${STORAGE_ACCOUNT_OWNER_USER_NAME}'","DataConsumerUsername":"'${DATA_CONSUMER_USER_NAME}'","ServiceUsersCredentials":'${SERVICE_USERS_CREDENTIALS}',"azureAccountName":"'${AZURE_ACCOUNT_NAME}'","azureAccountKey":"'${AZURE_SECRET_KEY}'","azureArchiveContainer":"'${AZURE_ARCHIVE_BUCKET_NAME}'","AzureArchiveAccessTier":"'${AZURE_ARCHIVE_ACCESS_TIER}'","azureArchiveManifestTier":"'${AZURE_ARCHIVE_MANIFEST_ACCESS_TIER}'","azureArchiveQueue":"'${AZURE_ARCHIVE_QUEUE_NAME}'","timeProgressionFactor":"'${TIME_PROGRESSION_FACTOR}'"}'
```

Comment:
this is becoming un-readable.... since it is a string containing JSON, we should be able to add empty lines, maybe?

so we could reformat this like (possibly even with keys sorted alphabetically), so it is easier to read and see the changes:

```suggestion
WORLD_PARAMETERS='{\
  "subdomain":"'${SUBDOMAIN}'",\
  "ssl":false,\
  "port":"'${ZENKO_PORT}'",\
  "AccountName":"'${ZENKO_ACCOUNT_NAME}'",\
  "AdminAccessKey":"'${ADMIN_ACCESS_KEY_ID}'",\
  "AdminSecretKey":"'${ADMIN_SECRET_ACCESS_KEY}'",\
  "VaultAuthHost":"'${VAULT_AUTH_HOST}'",\
  "NotificationDestination":"'${NOTIF_DEST_NAME}'",\ 
  "NotificationDestinationTopic":"'${NOTIF_DEST_TOPIC}'",\ 
  "NotificationDestinationAlt":"'${NOTIF_ALT_DEST_NAME}'",\
 "NotificationDestinationTopicAlt":"'${NOTIF_ALT_DEST_TOPIC}'",\
 "KafkaHosts":"'${KAFKA_HOST_PORT}'",\
 "KeycloakPassword":"'${KEYCLOAK_TEST_PASSWORD}'",\
 "KeycloakHost":"'${KEYCLOAK_TEST_HOST}'",\
 "KeycloakPort":"'${KEYCLOAK_TEST_PORT}'",\
 "keycloakRealm":"'${KEYCLOAK_TEST_REALM_NAME}'",\
 "keycloakClientId":"'${KEYCLOAK_TEST_CLIENT_ID}'",\
 "keycloakGrantType":"'${KEYCLOAK_TEST_GRANT_TYPE}'",\
 "StorageManagerUsername":"'${STORAGE_MANAGER_USER_NAME}'",\
 "StorageAccountOwnerUsername":"'${STORAGE_ACCOUNT_OWNER_USER_NAME}'",\
 "DataConsumerUsername":"'${DATA_CONSUMER_USER_NAME}'",\
 "ServiceUsersCredentials":'${SERVICE_USERS_CREDENTIALS}',\
 "azureAccountName":"'${AZURE_ACCOUNT_NAME}'",\
 "azureAccountKey":"'${AZURE_SECRET_KEY}'",\
 "azureArchiveContainer":"'${AZURE_ARCHIVE_BUCKET_NAME}'",\
 "AzureArchiveAccessTier":"'${AZURE_ARCHIVE_ACCESS_TIER}'",\
 "azureArchiveManifestTier":"'${AZURE_ARCHIVE_MANIFEST_ACCESS_TIER}'",\
 "azureArchiveQueue":"'${AZURE_ARCHIVE_QUEUE_NAME}'",\
 "timeProgressionFactor":"'${TIME_PROGRESSION_FACTOR}'"\
}'
```

Reply:
(and now we notice insonsitencies in the naming of keys 😄)